{"@t":"2023-05-09T22:03:23.8784858Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://mail.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":14760,"ProcessName":"w3wp","ThreadId":129,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2023-05-09T22:08:23.9239749Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://mail.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":14760,"ProcessName":"w3wp","ThreadId":244,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2023-05-09T22:13:23.9656772Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://mail.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":14760,"ProcessName":"w3wp","ThreadId":157,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2023-05-09T22:18:23.9975626Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://mail.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":14760,"ProcessName":"w3wp","ThreadId":171,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2023-05-09T22:23:24.0379562Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://mail.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":14760,"ProcessName":"w3wp","ThreadId":212,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2023-05-09T22:28:24.0721288Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://mail.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":14760,"ProcessName":"w3wp","ThreadId":212,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2023-05-09T22:33:24.1276390Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://mail.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":14760,"ProcessName":"w3wp","ThreadId":212,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2023-05-09T22:38:24.1772475Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://mail.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":14760,"ProcessName":"w3wp","ThreadId":99,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2023-05-09T22:43:24.2525101Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://mail.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":14760,"ProcessName":"w3wp","ThreadId":99,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2023-05-09T22:48:24.2883795Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://mail.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":14760,"ProcessName":"w3wp","ThreadId":104,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2023-05-09T22:53:24.3479660Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://mail.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":14760,"ProcessName":"w3wp","ThreadId":47,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2023-05-09T22:58:24.3999234Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://mail.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":14760,"ProcessName":"w3wp","ThreadId":47,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2023-05-09T23:03:24.4592200Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://mail.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":14760,"ProcessName":"w3wp","ThreadId":47,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2023-05-09T23:08:24.6088114Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://mail.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":14760,"ProcessName":"w3wp","ThreadId":72,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2023-05-09T23:13:24.6812328Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://mail.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":14760,"ProcessName":"w3wp","ThreadId":51,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2023-05-09T23:18:24.7216263Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://mail.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":14760,"ProcessName":"w3wp","ThreadId":157,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2023-05-09T23:23:24.7636341Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://mail.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":14760,"ProcessName":"w3wp","ThreadId":129,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2023-05-09T23:28:24.7920175Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://mail.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":14760,"ProcessName":"w3wp","ThreadId":80,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2023-05-09T23:33:24.8509288Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://mail.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":14760,"ProcessName":"w3wp","ThreadId":80,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2023-05-09T23:38:24.9100215Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://mail.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":14760,"ProcessName":"w3wp","ThreadId":80,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2023-05-09T23:43:24.9517281Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://mail.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":14760,"ProcessName":"w3wp","ThreadId":62,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2023-05-09T23:48:24.9867652Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://mail.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":14760,"ProcessName":"w3wp","ThreadId":52,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2023-05-09T23:53:25.0273463Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://mail.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":14760,"ProcessName":"w3wp","ThreadId":12,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2023-05-09T23:58:25.0611643Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://mail.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":14760,"ProcessName":"w3wp","ThreadId":191,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2023-05-10T00:03:25.0911550Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://mail.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":14760,"ProcessName":"w3wp","ThreadId":202,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2023-05-10T00:08:25.1264570Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://mail.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":14760,"ProcessName":"w3wp","ThreadId":202,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2023-05-10T00:13:25.1793555Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://mail.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":14760,"ProcessName":"w3wp","ThreadId":215,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2023-05-10T00:18:25.2160583Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://mail.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":14760,"ProcessName":"w3wp","ThreadId":215,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2023-05-10T00:23:25.2580642Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://mail.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":14760,"ProcessName":"w3wp","ThreadId":62,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2023-05-10T00:28:25.2969076Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://mail.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":14760,"ProcessName":"w3wp","ThreadId":174,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2023-05-10T00:33:25.3502895Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://mail.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":14760,"ProcessName":"w3wp","ThreadId":174,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2023-05-10T00:38:25.3965118Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://mail.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":14760,"ProcessName":"w3wp","ThreadId":91,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2023-05-10T00:43:25.4304634Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://mail.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":14760,"ProcessName":"w3wp","ThreadId":174,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2023-05-10T00:48:25.4679348Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://mail.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":14760,"ProcessName":"w3wp","ThreadId":174,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2023-05-10T00:53:25.4986482Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://mail.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":14760,"ProcessName":"w3wp","ThreadId":80,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2023-05-10T00:58:25.5414514Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://mail.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":14760,"ProcessName":"w3wp","ThreadId":80,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2023-05-10T01:03:25.5912408Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://mail.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":14760,"ProcessName":"w3wp","ThreadId":171,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2023-05-10T01:08:25.6411280Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://mail.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":14760,"ProcessName":"w3wp","ThreadId":171,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2023-05-10T01:13:25.6775572Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://mail.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":14760,"ProcessName":"w3wp","ThreadId":124,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2023-05-10T01:18:25.7144874Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://mail.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":14760,"ProcessName":"w3wp","ThreadId":124,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2023-05-10T01:23:25.7622646Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://mail.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":14760,"ProcessName":"w3wp","ThreadId":124,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2023-05-10T01:28:25.7899631Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://mail.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":14760,"ProcessName":"w3wp","ThreadId":48,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2023-05-10T01:33:25.8247477Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://mail.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":14760,"ProcessName":"w3wp","ThreadId":171,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2023-05-10T01:38:25.8726521Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://mail.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":14760,"ProcessName":"w3wp","ThreadId":216,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2023-05-10T01:43:25.9034904Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://mail.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":14760,"ProcessName":"w3wp","ThreadId":216,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2023-05-10T01:48:25.9469636Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://mail.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":14760,"ProcessName":"w3wp","ThreadId":216,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2023-05-10T01:53:25.9758325Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://mail.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":14760,"ProcessName":"w3wp","ThreadId":41,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2023-05-10T01:58:26.0185150Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://mail.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":14760,"ProcessName":"w3wp","ThreadId":21,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2023-05-10T02:03:26.0635847Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://mail.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":14760,"ProcessName":"w3wp","ThreadId":79,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2023-05-10T02:08:26.1129606Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://mail.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":14760,"ProcessName":"w3wp","ThreadId":79,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2023-05-10T02:13:26.1500774Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://mail.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":14760,"ProcessName":"w3wp","ThreadId":52,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2023-05-10T02:18:26.2074883Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://mail.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":14760,"ProcessName":"w3wp","ThreadId":31,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2023-05-10T02:23:26.2433161Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://mail.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":14760,"ProcessName":"w3wp","ThreadId":31,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2023-05-10T02:28:26.2901562Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://mail.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":14760,"ProcessName":"w3wp","ThreadId":16,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2023-05-10T02:33:26.3219970Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://mail.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":14760,"ProcessName":"w3wp","ThreadId":277,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2023-05-10T02:38:26.3637009Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://mail.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":14760,"ProcessName":"w3wp","ThreadId":31,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2023-05-10T02:43:26.4171605Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://mail.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":14760,"ProcessName":"w3wp","ThreadId":52,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2023-05-10T02:48:26.4543584Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://mail.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":14760,"ProcessName":"w3wp","ThreadId":52,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2023-05-10T02:53:26.5142455Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://mail.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":14760,"ProcessName":"w3wp","ThreadId":201,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2023-05-10T02:58:26.5434022Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://mail.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":14760,"ProcessName":"w3wp","ThreadId":201,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2023-05-10T03:03:26.6017300Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://mail.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":14760,"ProcessName":"w3wp","ThreadId":20,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2023-05-10T03:08:26.6718195Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://mail.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":14760,"ProcessName":"w3wp","ThreadId":51,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2023-05-10T03:13:26.7289945Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://mail.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":14760,"ProcessName":"w3wp","ThreadId":216,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2023-05-10T03:18:26.7767193Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://mail.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":14760,"ProcessName":"w3wp","ThreadId":212,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2023-05-10T03:23:26.8487230Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://mail.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":14760,"ProcessName":"w3wp","ThreadId":59,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2023-05-10T03:28:26.8922150Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://mail.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":14760,"ProcessName":"w3wp","ThreadId":289,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2023-05-10T03:33:26.9407801Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://mail.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":14760,"ProcessName":"w3wp","ThreadId":203,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2023-05-10T03:37:22.2771037Z","@mt":"Failed (will repeat).","@l":"Error","@x":"System.Data.SqlClient.SqlException (0x80131904): SHUTDOWN is in progress.\r\nLogin failed for user 'mentorplan_dk'.\r\nCannot continue the execution because the session is in the kill state.\r\nA severe error occurred on the current command.  The results, if any, should be discarded.\r\n   at System.Data.SqlClient.SqlConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)\r\n   at System.Data.SqlClient.SqlInternalConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)\r\n   at System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)\r\n   at System.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)\r\n   at System.Data.SqlClient.TdsParser.Run(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj)\r\n   at System.Data.SqlClient.TdsParser.TdsExecuteTransactionManagerRequest(Byte[] buffer, TransactionManagerRequestType request, String transactionName, TransactionManagerIsolationLevel isoLevel, Int32 timeout, SqlInternalTransaction transaction, TdsParserStateObject stateObj, Boolean isDelegateControlRequest)\r\n   at System.Data.SqlClient.SqlInternalConnectionTds.ExecuteTransactionYukon(TransactionRequest transactionRequest, String transactionName, IsolationLevel iso, SqlInternalTransaction internalTransaction, Boolean isDelegateControlRequest)\r\n   at System.Data.SqlClient.SqlInternalConnectionTds.ExecuteTransaction(TransactionRequest transactionRequest, String name, IsolationLevel iso, SqlInternalTransaction internalTransaction, Boolean isDelegateControlRequest)\r\n   at System.Data.SqlClient.SqlInternalConnection.BeginSqlTransaction(IsolationLevel iso, String transactionName, Boolean shouldReconnect)\r\n   at System.Data.SqlClient.SqlConnection.BeginTransaction(IsolationLevel iso, String transactionName)\r\n   at System.Data.SqlClient.SqlConnection.BeginDbTransaction(IsolationLevel isolationLevel)\r\n   at System.Data.Common.DbConnection.BeginTransaction(IsolationLevel isolationLevel)\r\n   at StackExchange.Profiling.Data.ProfiledDbConnection.BeginDbTransaction(IsolationLevel isolationLevel) in C:\\projects\\dotnet\\src\\MiniProfiler.Shared\\Data\\ProfiledDbConnection.cs:line 138\r\n   at System.Data.Common.DbConnection.BeginTransaction(IsolationLevel isolationLevel)\r\n   at Umbraco.Core.Persistence.FaultHandling.RetryDbConnection.BeginDbTransaction(IsolationLevel isolationLevel)\r\n   at System.Data.Common.DbConnection.BeginTransaction(IsolationLevel isolationLevel)\r\n   at NPoco.Database.BeginTransaction(IsolationLevel isolationLevel)\r\n   at Umbraco.Core.Scoping.Scope.get_Database()\r\n   at Umbraco.Core.Sync.DatabaseServerMessenger.Sync()\r\n   at Umbraco.Web.Compose.DatabaseServerRegistrarAndMessengerComponent.InstructionProcessTask.PerformRun()\r\nClientConnectionId:8ff11932-bcf9-43b3-9a9d-db4619a6758e\r\nError Number:6005,State:1,Class:14","SourceContext":"Umbraco.Web.Compose.DatabaseServerRegistrarAndMessengerComponent+InstructionProcessTask","ProcessId":14760,"ProcessName":"w3wp","ThreadId":88,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2023-05-10T03:37:54.3866895Z","@mt":"Failed (will repeat).","@l":"Error","@x":"System.Data.SqlClient.SqlException (0x80131904): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server) ---> System.ComponentModel.Win32Exception (0x80004005): The network path was not found\r\n   at System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling, SqlAuthenticationProviderManager sqlAuthProviderManager)\r\n   at System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)\r\n   at System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)\r\n   at System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)\r\n   at System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)\r\n   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)\r\n   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)\r\n   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)\r\n   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)\r\n   at System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)\r\n   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)\r\n   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)\r\n   at System.Data.SqlClient.SqlConnection.Open()\r\n   at NPoco.Database.OpenSharedConnectionImp(Boolean isInternal)\r\n   at NPoco.Database.BeginTransaction(IsolationLevel isolationLevel)\r\n   at Umbraco.Core.Scoping.Scope.get_Database()\r\n   at Umbraco.Core.Sync.DatabaseServerMessenger.Sync()\r\n   at Umbraco.Web.Compose.DatabaseServerRegistrarAndMessengerComponent.InstructionProcessTask.PerformRun()\r\nClientConnectionId:00000000-0000-0000-0000-000000000000\r\nError Number:53,State:0,Class:20","SourceContext":"Umbraco.Web.Compose.DatabaseServerRegistrarAndMessengerComponent+InstructionProcessTask","ProcessId":14760,"ProcessName":"w3wp","ThreadId":194,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2023-05-10T03:37:54.3866895Z","@mt":"Failed to update server record in database.","@l":"Error","@x":"System.Data.SqlClient.SqlException (0x80131904): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server) ---> System.ComponentModel.Win32Exception (0x80004005): The network path was not found\r\n   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)\r\n   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)\r\n   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)\r\n   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)\r\n   at System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)\r\n   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)\r\n   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)\r\n   at System.Data.SqlClient.SqlConnection.Open()\r\n   at NPoco.Database.OpenSharedConnectionImp(Boolean isInternal)\r\n   at NPoco.Database.BeginTransaction(IsolationLevel isolationLevel)\r\n   at Umbraco.Core.Scoping.Scope.get_Database()\r\n   at Umbraco.Core.Persistence.Repositories.Implement.ServerRegistrationRepository.PerformGetAll(Int32[] ids)\r\n   at Umbraco.Core.Cache.NoCacheRepositoryCachePolicy`2.GetAll(TId[] ids, Func`2 performGetAll)\r\n   at Umbraco.Core.Persistence.Repositories.Implement.RepositoryBase`2.GetMany(TId[] ids)\r\n   at Umbraco.Core.Services.Implement.ServerRegistrationService.TouchServer(String serverAddress, String serverIdentity, TimeSpan staleTimeout)\r\n   at Umbraco.Web.Compose.DatabaseServerRegistrarAndMessengerComponent.TouchServerTask.PerformRun()\r\nClientConnectionId:00000000-0000-0000-0000-000000000000\r\nError Number:53,State:0,Class:20","SourceContext":"Umbraco.Web.Compose.DatabaseServerRegistrarAndMessengerComponent","ProcessId":14760,"ProcessName":"w3wp","ThreadId":115,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2023-05-10T03:38:26.9740903Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://mail.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":14760,"ProcessName":"w3wp","ThreadId":30,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2023-05-10T03:43:27.3498119Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://mail.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":14760,"ProcessName":"w3wp","ThreadId":118,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2023-05-10T03:48:27.3842447Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://mail.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":14760,"ProcessName":"w3wp","ThreadId":192,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2023-05-10T03:53:27.4373615Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://mail.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":14760,"ProcessName":"w3wp","ThreadId":263,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2023-05-10T03:58:27.4814454Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://mail.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":14760,"ProcessName":"w3wp","ThreadId":104,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2023-05-10T04:03:27.5270230Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel. ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.\r\n   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)\r\n   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()","keepAlivePingUrl":"http://mail.mentorplan.dk/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":14760,"ProcessName":"w3wp","ThreadId":202,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"ERROR"}
{"@t":"2023-05-10T04:04:26.3225915Z","@mt":"{LogPrefix} Terminating {Immediate}","LogPrefix":"[HealthCheckNotifier] ","Immediate":"","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":14760,"ProcessName":"w3wp","ThreadId":88,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:04:26.3225915Z","@mt":"{LogPrefix} Terminating {Immediate}","LogPrefix":"[TempFileCleanup] ","Immediate":"","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":14760,"ProcessName":"w3wp","ThreadId":88,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:04:26.3225915Z","@mt":"{LogPrefix} Terminating {Immediate}","LogPrefix":"[LogScrubber] ","Immediate":"","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":14760,"ProcessName":"w3wp","ThreadId":88,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:04:26.3225915Z","@mt":"{LogPrefix} Terminating {Immediate}","LogPrefix":"[ScheduledTasks] ","Immediate":"","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":14760,"ProcessName":"w3wp","ThreadId":88,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:04:26.3225915Z","@mt":"{LogPrefix} Terminating {Immediate}","LogPrefix":"[Umbraco.Web.Scheduling.IBackgroundTask] ","Immediate":"","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":14760,"ProcessName":"w3wp","ThreadId":88,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:04:26.3225915Z","@mt":"{LogPrefix} Terminating {Immediate}","LogPrefix":"[KeepAlive] ","Immediate":"","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":14760,"ProcessName":"w3wp","ThreadId":88,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:04:26.3225915Z","@mt":"{LogPrefix} Terminating {Immediate}","LogPrefix":"[RebuildIndexesOnStartup] ","Immediate":"","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":14760,"ProcessName":"w3wp","ThreadId":88,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:04:26.3225915Z","@mt":"{LogPrefix} Terminating {Immediate}","LogPrefix":"[ServerRegistration] ","Immediate":"","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":14760,"ProcessName":"w3wp","ThreadId":88,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:04:26.3225915Z","@mt":"{LogPrefix} Terminating {Immediate}","LogPrefix":"[ScheduledPublishing] ","Immediate":"","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":14760,"ProcessName":"w3wp","ThreadId":88,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:04:26.3235897Z","@mt":"Stopping ({SignalSource})","SignalSource":"environment","SourceContext":"Umbraco.Core.Runtime.MainDom","ProcessId":14760,"ProcessName":"w3wp","ThreadId":88,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:04:26.3235897Z","@mt":"{StartMessage} [Timing {TimingId}]","StartMessage":"Examine shutting down","TimingId":"7ea8519","SourceContext":"Umbraco.Web.Search.ExamineComponent","ProcessId":14760,"ProcessName":"w3wp","ThreadId":88,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:04:26.3315894Z","@mt":"{LogPrefix} Canceling tasks","LogPrefix":"[HealthCheckNotifier] ","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":14760,"ProcessName":"w3wp","ThreadId":184,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:04:26.3325898Z","@mt":"{LogPrefix} Tasks {TaskStatus}, terminated","LogPrefix":"[HealthCheckNotifier] ","TaskStatus":"cancelled","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":14760,"ProcessName":"w3wp","ThreadId":184,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:04:26.3395928Z","@mt":"{LogPrefix} Canceling tasks","LogPrefix":"[TempFileCleanup] ","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":14760,"ProcessName":"w3wp","ThreadId":26,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:04:26.3395928Z","@mt":"{LogPrefix} Tasks {TaskStatus}, terminated","LogPrefix":"[TempFileCleanup] ","TaskStatus":"cancelled","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":14760,"ProcessName":"w3wp","ThreadId":26,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:04:26.3395928Z","@mt":"{LogPrefix} Canceling tasks","LogPrefix":"[ScheduledTasks] ","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":14760,"ProcessName":"w3wp","ThreadId":160,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:04:26.3395928Z","@mt":"{LogPrefix} Tasks {TaskStatus}, terminated","LogPrefix":"[ScheduledTasks] ","TaskStatus":"cancelled","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":14760,"ProcessName":"w3wp","ThreadId":160,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:04:26.3415911Z","@mt":"{EndMessage} ({Duration}ms) [Timing {TimingId}]","EndMessage":"Completed.","Duration":17,"TimingId":"7ea8519","SourceContext":"Umbraco.Web.Search.ExamineComponent","ProcessId":14760,"ProcessName":"w3wp","ThreadId":88,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:04:26.3685939Z","@mt":"{LogPrefix} Canceling tasks","LogPrefix":"[Umbraco.Web.Scheduling.IBackgroundTask] ","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":14760,"ProcessName":"w3wp","ThreadId":162,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:04:26.3685939Z","@mt":"{LogPrefix} Tasks {TaskStatus}, terminated","LogPrefix":"[Umbraco.Web.Scheduling.IBackgroundTask] ","TaskStatus":"cancelled","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":14760,"ProcessName":"w3wp","ThreadId":162,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:04:26.3685939Z","@mt":"{LogPrefix} Canceling tasks","LogPrefix":"[RebuildIndexesOnStartup] ","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":14760,"ProcessName":"w3wp","ThreadId":240,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:04:26.3685939Z","@mt":"{LogPrefix} Tasks {TaskStatus}, terminated","LogPrefix":"[RebuildIndexesOnStartup] ","TaskStatus":"cancelled","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":14760,"ProcessName":"w3wp","ThreadId":240,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:04:26.3685939Z","@mt":"{LogPrefix} Canceling tasks","LogPrefix":"[ServerRegistration] ","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":14760,"ProcessName":"w3wp","ThreadId":135,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:04:26.3685939Z","@mt":"{LogPrefix} Tasks {TaskStatus}, terminated","LogPrefix":"[ServerRegistration] ","TaskStatus":"cancelled","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":14760,"ProcessName":"w3wp","ThreadId":135,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:04:26.3695929Z","@mt":"{LogPrefix} Canceling tasks","LogPrefix":"[ScheduledPublishing] ","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":14760,"ProcessName":"w3wp","ThreadId":84,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:04:26.3695929Z","@mt":"{LogPrefix} Tasks {TaskStatus}, terminated","LogPrefix":"[ScheduledPublishing] ","TaskStatus":"cancelled","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":14760,"ProcessName":"w3wp","ThreadId":84,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:04:26.3695929Z","@mt":"{LogPrefix} Canceling tasks","LogPrefix":"[KeepAlive] ","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":14760,"ProcessName":"w3wp","ThreadId":84,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:04:26.3695929Z","@mt":"{LogPrefix} Tasks {TaskStatus}, terminated","LogPrefix":"[KeepAlive] ","TaskStatus":"cancelled","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":14760,"ProcessName":"w3wp","ThreadId":84,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:04:26.3695929Z","@mt":"{LogPrefix} Canceling tasks","LogPrefix":"[LogScrubber] ","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":14760,"ProcessName":"w3wp","ThreadId":84,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:04:26.3695929Z","@mt":"{LogPrefix} Tasks {TaskStatus}, terminated","LogPrefix":"[LogScrubber] ","TaskStatus":"cancelled","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":14760,"ProcessName":"w3wp","ThreadId":84,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:04:26.5496040Z","@mt":"Released from MainDom","SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":14760,"ProcessName":"w3wp","ThreadId":88,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:04:26.5496040Z","@mt":"Released ({SignalSource})","SignalSource":"environment","SourceContext":"Umbraco.Core.Runtime.MainDom","ProcessId":14760,"ProcessName":"w3wp","ThreadId":88,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:04:26.5506045Z","@mt":"{LogPrefix} Terminating {Immediate}","LogPrefix":"[TelemetryReporter] ","Immediate":"","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":14760,"ProcessName":"w3wp","ThreadId":88,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:04:26.5506045Z","@mt":"{LogPrefix} Terminating {Immediate}","LogPrefix":"[ServerInstProcess] ","Immediate":"","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":14760,"ProcessName":"w3wp","ThreadId":88,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:04:26.7656163Z","@mt":"{LogPrefix} Canceling tasks","LogPrefix":"[TelemetryReporter] ","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":14760,"ProcessName":"w3wp","ThreadId":259,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:04:26.7656163Z","@mt":"{LogPrefix} Tasks {TaskStatus}, terminated","LogPrefix":"[TelemetryReporter] ","TaskStatus":"cancelled","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":14760,"ProcessName":"w3wp","ThreadId":259,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:04:27.3676493Z","@mt":"{LogPrefix} Canceling tasks","LogPrefix":"[ServerInstProcess] ","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":14760,"ProcessName":"w3wp","ThreadId":194,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:04:27.3676493Z","@mt":"{LogPrefix} Tasks {TaskStatus}, terminated","LogPrefix":"[ServerInstProcess] ","TaskStatus":"cancelled","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":14760,"ProcessName":"w3wp","ThreadId":194,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:04:28.4258445Z","@mt":"{LogPrefix} Canceling tasks","LogPrefix":"[ScheduledPublishing] ","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":14760,"ProcessName":"w3wp","ThreadId":97,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:04:28.4258445Z","@mt":"{LogPrefix} Canceling tasks","LogPrefix":"[ServerRegistration] ","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":14760,"ProcessName":"w3wp","ThreadId":135,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:04:28.4258445Z","@mt":"{LogPrefix} Canceling tasks","LogPrefix":"[RebuildIndexesOnStartup] ","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":14760,"ProcessName":"w3wp","ThreadId":240,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:04:28.4258445Z","@mt":"{LogPrefix} Canceling tasks","LogPrefix":"[KeepAlive] ","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":14760,"ProcessName":"w3wp","ThreadId":63,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:04:28.4258445Z","@mt":"{LogPrefix} Canceling tasks","LogPrefix":"[Umbraco.Web.Scheduling.IBackgroundTask] ","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":14760,"ProcessName":"w3wp","ThreadId":162,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:04:28.4883404Z","@mt":"{LogPrefix} Canceling tasks","LogPrefix":"[HealthCheckNotifier] ","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":14760,"ProcessName":"w3wp","ThreadId":225,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:04:28.4883404Z","@mt":"{LogPrefix} Canceling tasks","LogPrefix":"[TempFileCleanup] ","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":14760,"ProcessName":"w3wp","ThreadId":184,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:04:28.5977142Z","@mt":"{LogPrefix} Canceling tasks","LogPrefix":"[ScheduledTasks] ","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":14760,"ProcessName":"w3wp","ThreadId":160,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:04:28.5977142Z","@mt":"{LogPrefix} Canceling tasks","LogPrefix":"[LogScrubber] ","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":14760,"ProcessName":"w3wp","ThreadId":26,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:04:28.7383401Z","@mt":"{LogPrefix} Canceling tasks","LogPrefix":"[TelemetryReporter] ","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":14760,"ProcessName":"w3wp","ThreadId":84,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:04:28.9102195Z","@mt":"{LogPrefix} Canceling tasks","LogPrefix":"[ServerInstProcess] ","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":14760,"ProcessName":"w3wp","ThreadId":259,"AppDomainId":3,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:22:08.7499806Z","@mt":"{StartMessage} [Timing {TimingId}]","StartMessage":"Booting Umbraco 8.17.0.","TimingId":"14c9479","SourceContext":"Umbraco.Core.Runtime.CoreRuntime","ProcessId":18936,"ProcessName":"w3wp","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"4bb470d3-25a4-4ead-b2dc-6b40c8e694f2"}
{"@t":"2023-05-10T04:22:08.7656141Z","@mt":"Booting site '{HostingSiteName}', app '{HostingApplicationID}', path '{HostingPhysicalPath}', server '{MachineName}'.","HostingSiteName":"mentorplan.dk","HostingApplicationID":"/LM/W3SVC/844263/ROOT","HostingPhysicalPath":"d:\\web\\localuser\\mentorplan.dk\\public_html\\","MachineName":"NT45","SourceContext":"Umbraco.Core.Runtime.CoreRuntime","ProcessId":18936,"ProcessName":"w3wp","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"4bb470d3-25a4-4ead-b2dc-6b40c8e694f2"}
{"@t":"2023-05-10T04:22:08.8125115Z","@mt":"Acquiring.","SourceContext":"Umbraco.Core.Runtime.MainDom","ProcessId":18936,"ProcessName":"w3wp","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"4bb470d3-25a4-4ead-b2dc-6b40c8e694f2"}
{"@t":"2023-05-10T04:22:08.8125115Z","@mt":"Acquired.","SourceContext":"Umbraco.Core.Runtime.MainDom","ProcessId":18936,"ProcessName":"w3wp","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"4bb470d3-25a4-4ead-b2dc-6b40c8e694f2"}
{"@t":"2023-05-10T04:22:09.3060342Z","@mt":"{StartMessage} [Timing {TimingId}]","StartMessage":"Resolving composer types.","TimingId":"28a1912","SourceContext":"Umbraco.Core.Runtime.CoreRuntime","ProcessId":18936,"ProcessName":"w3wp","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"4bb470d3-25a4-4ead-b2dc-6b40c8e694f2"}
{"@t":"2023-05-10T04:22:09.3216699Z","@mt":"{EndMessage} ({Duration}ms) [Timing {TimingId}]","EndMessage":"Resolved.","Duration":24,"TimingId":"28a1912","SourceContext":"Umbraco.Core.Runtime.CoreRuntime","ProcessId":18936,"ProcessName":"w3wp","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"4bb470d3-25a4-4ead-b2dc-6b40c8e694f2"}
{"@t":"2023-05-10T04:22:10.1250020Z","@mt":"Profiler is VoidProfiler, not profiling (must run debug mode to profile).","SourceContext":"Umbraco.Web.Logging.WebProfilerComponent","ProcessId":18936,"ProcessName":"w3wp","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"4bb470d3-25a4-4ead-b2dc-6b40c8e694f2"}
{"@t":"2023-05-10T04:22:10.1718965Z","@mt":"Initializing Umbraco internal event handlers for cache refreshing.","SourceContext":"Umbraco.Web.Cache.DistributedCacheBinderComponent","ProcessId":18936,"ProcessName":"w3wp","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"4bb470d3-25a4-4ead-b2dc-6b40c8e694f2"}
{"@t":"2023-05-10T04:22:10.2344277Z","@mt":"Adding examine event handlers for {RegisteredIndexers} index providers.","RegisteredIndexers":3,"SourceContext":"Umbraco.Web.Search.ExamineComponent","ProcessId":18936,"ProcessName":"w3wp","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"4bb470d3-25a4-4ead-b2dc-6b40c8e694f2"}
{"@t":"2023-05-10T04:22:10.2344277Z","@mt":"{EndMessage} ({Duration}ms) [Timing {TimingId}]","EndMessage":"Booted.","Duration":1482,"TimingId":"14c9479","SourceContext":"Umbraco.Core.Runtime.CoreRuntime","ProcessId":18936,"ProcessName":"w3wp","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"4bb470d3-25a4-4ead-b2dc-6b40c8e694f2"}
{"@t":"2023-05-10T04:22:10.4741021Z","@mt":"New url {Url} detected, re-discovering application url.","Url":"http://mentorplan.dk/umbraco","SourceContext":"Umbraco.Core.Sync.ApplicationUrlHelper","ProcessId":18936,"ProcessName":"w3wp","ThreadId":11,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO ","HttpRequestNumber":2,"HttpRequestId":"e0348312-29fd-4c6d-8455-a82af92c80de"}
{"@t":"2023-05-10T04:22:10.4751030Z","@mt":"ApplicationUrl: {UmbracoAppUrl} (UmbracoModule request)","UmbracoAppUrl":"http://mentorplan.dk/umbraco","SourceContext":"Umbraco.Core.Sync.ApplicationUrlHelper","ProcessId":18936,"ProcessName":"w3wp","ThreadId":11,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO ","HttpRequestNumber":2,"HttpRequestId":"e0348312-29fd-4c6d-8455-a82af92c80de"}
{"@t":"2023-05-10T04:57:08.8725684Z","@mt":"{LogPrefix} Terminating {Immediate}","LogPrefix":"[ServerRegistration] ","Immediate":"","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":18936,"ProcessName":"w3wp","ThreadId":10,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:57:08.8725684Z","@mt":"{LogPrefix} Canceling tasks","LogPrefix":"[ServerRegistration] ","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":18936,"ProcessName":"w3wp","ThreadId":32,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:57:08.8725684Z","@mt":"Stopping ({SignalSource})","SignalSource":"environment","SourceContext":"Umbraco.Core.Runtime.MainDom","ProcessId":18936,"ProcessName":"w3wp","ThreadId":10,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:57:08.8725684Z","@mt":"{LogPrefix} Tasks {TaskStatus}, terminated","LogPrefix":"[ServerRegistration] ","TaskStatus":"cancelled","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":18936,"ProcessName":"w3wp","ThreadId":32,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:57:08.8725684Z","@mt":"{StartMessage} [Timing {TimingId}]","StartMessage":"Examine shutting down","TimingId":"551815b","SourceContext":"Umbraco.Web.Search.ExamineComponent","ProcessId":18936,"ProcessName":"w3wp","ThreadId":10,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:57:08.8725684Z","@mt":"{EndMessage} ({Duration}ms) [Timing {TimingId}]","EndMessage":"Completed.","Duration":4,"TimingId":"551815b","SourceContext":"Umbraco.Web.Search.ExamineComponent","ProcessId":18936,"ProcessName":"w3wp","ThreadId":10,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:57:08.8882044Z","@mt":"Released ({SignalSource})","SignalSource":"environment","SourceContext":"Umbraco.Core.Runtime.MainDom","ProcessId":18936,"ProcessName":"w3wp","ThreadId":10,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:57:08.8882044Z","@mt":"{LogPrefix} Terminating {Immediate}","LogPrefix":"[LogScrubber] ","Immediate":"","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":18936,"ProcessName":"w3wp","ThreadId":10,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:57:08.8882044Z","@mt":"{LogPrefix} Terminating {Immediate}","LogPrefix":"[ScheduledPublishing] ","Immediate":"","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":18936,"ProcessName":"w3wp","ThreadId":10,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:57:08.8882044Z","@mt":"{LogPrefix} Terminating {Immediate}","LogPrefix":"[KeepAlive] ","Immediate":"","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":18936,"ProcessName":"w3wp","ThreadId":10,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:57:08.8882044Z","@mt":"{LogPrefix} Terminating {Immediate}","LogPrefix":"[ScheduledTasks] ","Immediate":"","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":18936,"ProcessName":"w3wp","ThreadId":10,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:57:08.8882044Z","@mt":"{LogPrefix} Terminating {Immediate}","LogPrefix":"[TelemetryReporter] ","Immediate":"","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":18936,"ProcessName":"w3wp","ThreadId":10,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:57:08.8882044Z","@mt":"{LogPrefix} Terminating {Immediate}","LogPrefix":"[HealthCheckNotifier] ","Immediate":"","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":18936,"ProcessName":"w3wp","ThreadId":10,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:57:08.8882044Z","@mt":"{LogPrefix} Terminating {Immediate}","LogPrefix":"[ServerInstProcess] ","Immediate":"","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":18936,"ProcessName":"w3wp","ThreadId":10,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:57:08.8884733Z","@mt":"{LogPrefix} Canceling tasks","LogPrefix":"[ScheduledPublishing] ","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":18936,"ProcessName":"w3wp","ThreadId":30,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:57:08.8884733Z","@mt":"{LogPrefix} Tasks {TaskStatus}, terminated","LogPrefix":"[ScheduledPublishing] ","TaskStatus":"cancelled","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":18936,"ProcessName":"w3wp","ThreadId":30,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:57:08.8884733Z","@mt":"{LogPrefix} Canceling tasks","LogPrefix":"[LogScrubber] ","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":18936,"ProcessName":"w3wp","ThreadId":31,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:57:08.8884733Z","@mt":"{LogPrefix} Tasks {TaskStatus}, terminated","LogPrefix":"[LogScrubber] ","TaskStatus":"cancelled","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":18936,"ProcessName":"w3wp","ThreadId":31,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:57:08.8894667Z","@mt":"{LogPrefix} Terminating {Immediate}","LogPrefix":"[TempFileCleanup] ","Immediate":"","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":18936,"ProcessName":"w3wp","ThreadId":10,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:57:08.8894667Z","@mt":"{LogPrefix} Terminating {Immediate}","LogPrefix":"[Umbraco.Web.Scheduling.IBackgroundTask] ","Immediate":"","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":18936,"ProcessName":"w3wp","ThreadId":10,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:57:08.8894667Z","@mt":"{LogPrefix} Canceling tasks","LogPrefix":"[KeepAlive] ","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":18936,"ProcessName":"w3wp","ThreadId":26,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:57:08.8894667Z","@mt":"{LogPrefix} Tasks {TaskStatus}, terminated","LogPrefix":"[KeepAlive] ","TaskStatus":"cancelled","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":18936,"ProcessName":"w3wp","ThreadId":26,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:57:08.8904673Z","@mt":"{LogPrefix} Canceling tasks","LogPrefix":"[ScheduledTasks] ","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":18936,"ProcessName":"w3wp","ThreadId":21,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:57:08.8904673Z","@mt":"{LogPrefix} Tasks {TaskStatus}, terminated","LogPrefix":"[ScheduledTasks] ","TaskStatus":"cancelled","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":18936,"ProcessName":"w3wp","ThreadId":21,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:57:08.8904673Z","@mt":"{LogPrefix} Canceling tasks","LogPrefix":"[HealthCheckNotifier] ","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":18936,"ProcessName":"w3wp","ThreadId":29,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:57:08.8904673Z","@mt":"{LogPrefix} Tasks {TaskStatus}, terminated","LogPrefix":"[HealthCheckNotifier] ","TaskStatus":"cancelled","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":18936,"ProcessName":"w3wp","ThreadId":29,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:57:08.8904673Z","@mt":"{LogPrefix} Canceling tasks","LogPrefix":"[ServerInstProcess] ","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":18936,"ProcessName":"w3wp","ThreadId":38,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:57:08.8904673Z","@mt":"{LogPrefix} Tasks {TaskStatus}, terminated","LogPrefix":"[ServerInstProcess] ","TaskStatus":"cancelled","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":18936,"ProcessName":"w3wp","ThreadId":38,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:57:08.8914675Z","@mt":"{LogPrefix} Canceling tasks","LogPrefix":"[TempFileCleanup] ","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":18936,"ProcessName":"w3wp","ThreadId":34,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:57:08.8914675Z","@mt":"{LogPrefix} Tasks {TaskStatus}, terminated","LogPrefix":"[TempFileCleanup] ","TaskStatus":"cancelled","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":18936,"ProcessName":"w3wp","ThreadId":34,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:57:08.8914675Z","@mt":"{LogPrefix} Canceling tasks","LogPrefix":"[Umbraco.Web.Scheduling.IBackgroundTask] ","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":18936,"ProcessName":"w3wp","ThreadId":36,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:57:08.8914675Z","@mt":"{LogPrefix} Tasks {TaskStatus}, terminated","LogPrefix":"[Umbraco.Web.Scheduling.IBackgroundTask] ","TaskStatus":"cancelled","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":18936,"ProcessName":"w3wp","ThreadId":36,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:57:08.8925080Z","@mt":"{LogPrefix} Canceling tasks","LogPrefix":"[TelemetryReporter] ","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":18936,"ProcessName":"w3wp","ThreadId":41,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:57:08.8925080Z","@mt":"{LogPrefix} Tasks {TaskStatus}, terminated","LogPrefix":"[TelemetryReporter] ","TaskStatus":"cancelled","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":18936,"ProcessName":"w3wp","ThreadId":41,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T04:57:09.0133902Z","@mt":"Application shutdown. Details: {ShutdownReason}\r\n\r\n_shutDownMessage={ShutdownMessage}\r\n\r\n_shutDownStack={ShutdownStack}","ShutdownReason":"HostingEnvironment","ShutdownMessage":"HostingEnvironment initiated shutdown\r\nHostingEnvironment caused shutdown","ShutdownStack":"   at System.Environment.GetStackTrace(Exception e, Boolean needFileInfo)\r\n   at System.Environment.get_StackTrace()\r\n   at System.Web.Hosting.HostingEnvironment.InitiateShutdownInternal()\r\n   at System.Web.Hosting.HostingEnvironment.InitiateShutdownWithoutDemand()\r\n   at System.Web.Hosting.PipelineRuntime.StopProcessing()","SourceContext":"Umbraco.Web.UmbracoApplicationBase","ProcessId":18936,"ProcessName":"w3wp","ThreadId":40,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T06:23:19.5311841Z","@mt":"{StartMessage} [Timing {TimingId}]","StartMessage":"Booting Umbraco 8.17.0.","TimingId":"38df59a","SourceContext":"Umbraco.Core.Runtime.CoreRuntime","ProcessId":23040,"ProcessName":"w3wp","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"f9b942b9-f37d-4ce8-9fd0-a59b9dcde53c"}
{"@t":"2023-05-10T06:23:19.5468084Z","@mt":"Booting site '{HostingSiteName}', app '{HostingApplicationID}', path '{HostingPhysicalPath}', server '{MachineName}'.","HostingSiteName":"mentorplan.dk","HostingApplicationID":"/LM/W3SVC/844263/ROOT","HostingPhysicalPath":"d:\\web\\localuser\\mentorplan.dk\\public_html\\","MachineName":"NT45","SourceContext":"Umbraco.Core.Runtime.CoreRuntime","ProcessId":23040,"ProcessName":"w3wp","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"f9b942b9-f37d-4ce8-9fd0-a59b9dcde53c"}
{"@t":"2023-05-10T06:23:19.5780611Z","@mt":"Acquiring.","SourceContext":"Umbraco.Core.Runtime.MainDom","ProcessId":23040,"ProcessName":"w3wp","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"f9b942b9-f37d-4ce8-9fd0-a59b9dcde53c"}
{"@t":"2023-05-10T06:23:19.5780611Z","@mt":"Acquired.","SourceContext":"Umbraco.Core.Runtime.MainDom","ProcessId":23040,"ProcessName":"w3wp","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"f9b942b9-f37d-4ce8-9fd0-a59b9dcde53c"}
{"@t":"2023-05-10T06:23:19.9581432Z","@mt":"{StartMessage} [Timing {TimingId}]","StartMessage":"Resolving composer types.","TimingId":"a9ed22a","SourceContext":"Umbraco.Core.Runtime.CoreRuntime","ProcessId":23040,"ProcessName":"w3wp","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"f9b942b9-f37d-4ce8-9fd0-a59b9dcde53c"}
{"@t":"2023-05-10T06:23:19.9737690Z","@mt":"{EndMessage} ({Duration}ms) [Timing {TimingId}]","EndMessage":"Resolved.","Duration":15,"TimingId":"a9ed22a","SourceContext":"Umbraco.Core.Runtime.CoreRuntime","ProcessId":23040,"ProcessName":"w3wp","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"f9b942b9-f37d-4ce8-9fd0-a59b9dcde53c"}
{"@t":"2023-05-10T06:23:20.5420973Z","@mt":"Profiler is VoidProfiler, not profiling (must run debug mode to profile).","SourceContext":"Umbraco.Web.Logging.WebProfilerComponent","ProcessId":23040,"ProcessName":"w3wp","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"f9b942b9-f37d-4ce8-9fd0-a59b9dcde53c"}
{"@t":"2023-05-10T06:23:20.5889553Z","@mt":"Initializing Umbraco internal event handlers for cache refreshing.","SourceContext":"Umbraco.Web.Cache.DistributedCacheBinderComponent","ProcessId":23040,"ProcessName":"w3wp","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"f9b942b9-f37d-4ce8-9fd0-a59b9dcde53c"}
{"@t":"2023-05-10T06:23:20.6358372Z","@mt":"Adding examine event handlers for {RegisteredIndexers} index providers.","RegisteredIndexers":3,"SourceContext":"Umbraco.Web.Search.ExamineComponent","ProcessId":23040,"ProcessName":"w3wp","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"f9b942b9-f37d-4ce8-9fd0-a59b9dcde53c"}
{"@t":"2023-05-10T06:23:20.6358372Z","@mt":"{EndMessage} ({Duration}ms) [Timing {TimingId}]","EndMessage":"Booted.","Duration":1108,"TimingId":"38df59a","SourceContext":"Umbraco.Core.Runtime.CoreRuntime","ProcessId":23040,"ProcessName":"w3wp","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"f9b942b9-f37d-4ce8-9fd0-a59b9dcde53c"}
{"@t":"2023-05-10T06:23:20.8249388Z","@mt":"New url {Url} detected, re-discovering application url.","Url":"http://mentorplan.dk/umbraco","SourceContext":"Umbraco.Core.Sync.ApplicationUrlHelper","ProcessId":23040,"ProcessName":"w3wp","ThreadId":10,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO ","HttpRequestNumber":2,"HttpRequestId":"1476091f-2ff1-46fc-8baa-f64b67f79b16"}
{"@t":"2023-05-10T06:23:20.8405652Z","@mt":"ApplicationUrl: {UmbracoAppUrl} (UmbracoModule request)","UmbracoAppUrl":"http://mentorplan.dk/umbraco","SourceContext":"Umbraco.Core.Sync.ApplicationUrlHelper","ProcessId":23040,"ProcessName":"w3wp","ThreadId":10,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO ","HttpRequestNumber":2,"HttpRequestId":"1476091f-2ff1-46fc-8baa-f64b67f79b16"}
{"@t":"2023-05-10T06:23:21.3226591Z","@mt":"Registered with MainDom, localContentDbExists? {LocalContentDbExists}, localMediaDbExists? {LocalMediaDbExists}","LocalContentDbExists":true,"LocalMediaDbExists":true,"SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":23040,"ProcessName":"w3wp","ThreadId":10,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO ","HttpRequestNumber":2,"HttpRequestId":"1476091f-2ff1-46fc-8baa-f64b67f79b16"}
{"@t":"2023-05-10T06:23:21.3226591Z","@mt":"Creating the content store, localContentDbExists? {LocalContentDbExists}","LocalContentDbExists":true,"SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":23040,"ProcessName":"w3wp","ThreadId":10,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO ","HttpRequestNumber":2,"HttpRequestId":"1476091f-2ff1-46fc-8baa-f64b67f79b16"}
{"@t":"2023-05-10T06:23:21.3236568Z","@mt":"Creating the media store, localMediaDbExists? {LocalMediaDbExists}","LocalMediaDbExists":true,"SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":23040,"ProcessName":"w3wp","ThreadId":10,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO ","HttpRequestNumber":2,"HttpRequestId":"1476091f-2ff1-46fc-8baa-f64b67f79b16"}
{"@t":"2023-05-10T06:23:21.7123827Z","@mt":"{StartMessage} [Timing {TimingId}]","StartMessage":"Loading content from local cache file","TimingId":"5905a81","SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":23040,"ProcessName":"w3wp","ThreadId":10,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO ","HttpRequestNumber":2,"HttpRequestId":"1476091f-2ff1-46fc-8baa-f64b67f79b16"}
{"@t":"2023-05-10T06:23:21.7436383Z","@mt":"{EndMessage} ({Duration}ms) [Timing {TimingId}]","EndMessage":"Completed.","Duration":21,"TimingId":"5905a81","SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":23040,"ProcessName":"w3wp","ThreadId":10,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO ","HttpRequestNumber":2,"HttpRequestId":"1476091f-2ff1-46fc-8baa-f64b67f79b16"}
{"@t":"2023-05-10T06:23:21.7436383Z","@mt":"{StartMessage} [Timing {TimingId}]","StartMessage":"Loading media from local cache file","TimingId":"7a98427","SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":23040,"ProcessName":"w3wp","ThreadId":10,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO ","HttpRequestNumber":2,"HttpRequestId":"1476091f-2ff1-46fc-8baa-f64b67f79b16"}
{"@t":"2023-05-10T06:23:21.7436383Z","@mt":"{EndMessage} ({Duration}ms) [Timing {TimingId}]","EndMessage":"Completed.","Duration":0,"TimingId":"7a98427","SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":23040,"ProcessName":"w3wp","ThreadId":10,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO ","HttpRequestNumber":2,"HttpRequestId":"1476091f-2ff1-46fc-8baa-f64b67f79b16"}
{"@t":"2023-05-10T06:23:21.8530270Z","@mt":"Starting initialize async background thread.","SourceContext":"Umbraco.Web.Search.BackgroundIndexRebuilder","ProcessId":23040,"ProcessName":"w3wp","ThreadId":10,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO ","HttpRequestNumber":2,"HttpRequestId":"1476091f-2ff1-46fc-8baa-f64b67f79b16"}
{"@t":"2023-05-10T06:26:22.2616947Z","@mt":"Scheduled health check results:","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":23040,"ProcessName":"w3wp","ThreadId":24,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T06:26:22.2616947Z","@mt":"Checks for '{HealthCheckName}' all completed successfully.","HealthCheckName":"Folder & File Permissions","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":23040,"ProcessName":"w3wp","ThreadId":24,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T06:26:22.2616947Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Folder & File Permissions","HealthCheckResult":"Success","HealthCheckMessage":"All folders have the correct permissions set.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":23040,"ProcessName":"w3wp","ThreadId":24,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T06:26:22.2616947Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Folder & File Permissions","HealthCheckResult":"Success","HealthCheckMessage":"All files have the correct permissions set.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":23040,"ProcessName":"w3wp","ThreadId":24,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T06:26:22.2616947Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"SMTP Settings","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":23040,"ProcessName":"w3wp","ThreadId":24,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"WARN "}
{"@t":"2023-05-10T06:26:22.2616947Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"SMTP Settings","HealthCheckResult":"Error","HealthCheckMessage":"In the Web.config file system.net/mailsettings section, the host is not configured.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":23040,"ProcessName":"w3wp","ThreadId":24,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T06:26:22.2616947Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"Cross-site scripting Protection (X-XSS-Protection header)","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":23040,"ProcessName":"w3wp","ThreadId":24,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"WARN "}
{"@t":"2023-05-10T06:26:22.2616947Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Cross-site scripting Protection (X-XSS-Protection header)","HealthCheckResult":"Error","HealthCheckMessage":"The header <strong>X-XSS-Protection</strong> was not found.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":23040,"ProcessName":"w3wp","ThreadId":24,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T06:26:22.2616947Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"Cookie hijacking and protocol downgrade attacks Protection (Strict-Transport-Security Header (HSTS))","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":23040,"ProcessName":"w3wp","ThreadId":24,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"WARN "}
{"@t":"2023-05-10T06:26:22.2616947Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Cookie hijacking and protocol downgrade attacks Protection (Strict-Transport-Security Header (HSTS))","HealthCheckResult":"Error","HealthCheckMessage":"The header <strong>Strict-Transport-Security</strong> was not found.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":23040,"ProcessName":"w3wp","ThreadId":24,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T06:26:22.2616947Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"Content/MIME Sniffing Protection","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":23040,"ProcessName":"w3wp","ThreadId":24,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"WARN "}
{"@t":"2023-05-10T06:26:22.2616947Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Content/MIME Sniffing Protection","HealthCheckResult":"Error","HealthCheckMessage":"The header or meta-tag <strong>X-Content-Type-Options</strong> used to protect against MIME sniffing vulnerabilities was not found.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":23040,"ProcessName":"w3wp","ThreadId":24,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T06:26:22.2616947Z","@mt":"Checks for '{HealthCheckName}' all completed successfully.","HealthCheckName":"Excessive Headers","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":23040,"ProcessName":"w3wp","ThreadId":24,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T06:26:22.2616947Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Excessive Headers","HealthCheckResult":"Success","HealthCheckMessage":"No headers revealing information about the website technology were found.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":23040,"ProcessName":"w3wp","ThreadId":24,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T06:26:22.2616947Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"Click-Jacking Protection","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":23040,"ProcessName":"w3wp","ThreadId":24,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"WARN "}
{"@t":"2023-05-10T06:26:22.2616947Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Click-Jacking Protection","HealthCheckResult":"Error","HealthCheckMessage":"The header or meta-tag <strong>X-Frame-Options</strong> used to control whether a site can be IFRAMEd by another was not found.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":23040,"ProcessName":"w3wp","ThreadId":24,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T06:26:22.2616947Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"HTTPS Configuration","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":23040,"ProcessName":"w3wp","ThreadId":24,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"WARN "}
{"@t":"2023-05-10T06:26:22.2616947Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"HTTPS Configuration","HealthCheckResult":"Error","HealthCheckMessage":"You are currently not viewing the site using the HTTPS scheme.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":23040,"ProcessName":"w3wp","ThreadId":24,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T06:26:22.2616947Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"HTTPS Configuration","HealthCheckResult":"Info","HealthCheckMessage":"The appSetting 'Umbraco.Core.UseHttps' is set to 'false' in your web.config file. Once you access this site using the HTTPS scheme, that should be set to 'true'.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":23040,"ProcessName":"w3wp","ThreadId":24,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T06:26:22.2616947Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"HTTPS Configuration","HealthCheckResult":"Success","HealthCheckMessage":"Your website's certificate is valid.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":23040,"ProcessName":"w3wp","ThreadId":24,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T06:26:22.2616947Z","@mt":"Checks for '{HealthCheckName}' all completed successfully.","HealthCheckName":"Database data integrity check","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":23040,"ProcessName":"w3wp","ThreadId":24,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T06:26:22.2616947Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Database data integrity check","HealthCheckResult":"Success","HealthCheckMessage":"<p>All document paths are valid</p>\r\n","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":23040,"ProcessName":"w3wp","ThreadId":24,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T06:26:22.2616947Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Database data integrity check","HealthCheckResult":"Success","HealthCheckMessage":"<p>All media paths are valid</p>\r\n","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":23040,"ProcessName":"w3wp","ThreadId":24,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T06:26:22.2616947Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"Macro errors","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":23040,"ProcessName":"w3wp","ThreadId":24,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"WARN "}
{"@t":"2023-05-10T06:26:22.2616947Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Macro errors","HealthCheckResult":"Error","HealthCheckMessage":"MacroErrors are set to 'throw' which will prevent some or all pages in your site from loading completely if there are any errors in macros. Rectifying this will set the value to 'inline'.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":23040,"ProcessName":"w3wp","ThreadId":24,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T06:26:22.2616947Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"Notification Email Settings","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":23040,"ProcessName":"w3wp","ThreadId":24,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"WARN "}
{"@t":"2023-05-10T06:26:22.2616947Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Notification Email Settings","HealthCheckResult":"Error","HealthCheckMessage":"Notification email is still set to the default value of <strong>your@email.here</strong>.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":23040,"ProcessName":"w3wp","ThreadId":24,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T06:26:22.2616947Z","@mt":"Checks for '{HealthCheckName}' all completed successfully.","HealthCheckName":"Try Skip IIS Custom Errors","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":23040,"ProcessName":"w3wp","ThreadId":24,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T06:26:22.2616947Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Try Skip IIS Custom Errors","HealthCheckResult":"Success","HealthCheckMessage":"Try Skip IIS Custom Errors is set to 'true' and you're using IIS version '10.0'.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":23040,"ProcessName":"w3wp","ThreadId":24,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T06:26:22.2616947Z","@mt":"Checks for '{HealthCheckName}' all completed successfully.","HealthCheckName":"Custom Errors","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":23040,"ProcessName":"w3wp","ThreadId":24,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T06:26:22.2616947Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Custom Errors","HealthCheckResult":"Success","HealthCheckMessage":"Custom errors are set to 'RemoteOnly'.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":23040,"ProcessName":"w3wp","ThreadId":24,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T06:26:22.2616947Z","@mt":"Checks for '{HealthCheckName}' all completed successfully.","HealthCheckName":"Trace Mode","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":23040,"ProcessName":"w3wp","ThreadId":24,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T06:26:22.2616947Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Trace Mode","HealthCheckResult":"Success","HealthCheckMessage":"Trace mode is disabled.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":23040,"ProcessName":"w3wp","ThreadId":24,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T06:26:22.2616947Z","@mt":"Checks for '{HealthCheckName}' all completed successfully.","HealthCheckName":"Debug Compilation Mode","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":23040,"ProcessName":"w3wp","ThreadId":24,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
{"@t":"2023-05-10T06:26:22.2616947Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Debug Compilation Mode","HealthCheckResult":"Success","HealthCheckMessage":"Debug compilation mode is disabled.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":23040,"ProcessName":"w3wp","ThreadId":24,"AppDomainId":2,"AppDomainAppId":"LMW3SVC844263ROOT","MachineName":"NT45","Log4NetLevel":"INFO "}
